integrations/gtag/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@segment/analytics.js-integration-gtag",
+  "description": "The Gtag analytics.js integration.",
+  "version": "1.0.0",
```

Comment:
One minor change before this is finally merged in. Can we change this to `1.0.0-beta,`, please?